docs/man_pages/bodhi.rst

```diff
@@ -90,6 +90,21 @@ The ``overrides`` command allows users to manage build overrides.
 
         Notes on why this override is in place.
 
+``bodhi overrides edit [options] <nvr>``
+
+    Edit the build root given by ``<nvr>`` as a buildroot override. The ``edit`` subcommand supports
+    the following options:
+
+    ``--duration <days>``
+
+        The number of days the override should exist, given as an integer.
+
+    ``--notes <text>``
+
+        Notes on why this override is in place.
+
+    ``--expired``
+        Force an override to the expired state.
```

Comment:
Since these options are shared with the save command, maybe there's a way to document them once and just mention that they are shared by the edit and save subcommands?